src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController.php

```diff
@@ -1446,6 +1415,48 @@ public function getPricesAction(int $orderId): JsonResponse
         ]);
     }
 
+    /**
+     * @AdminSecurity("is_granted('read', request.get('_legacy_controller'))", redirectRoute="admin_orders_index")
+     *
+     * @param int $orderId
+     *
+     * @return Response
+     */
+    public function getProductsListAction(int $orderId): Response
+    {
+        /** @var OrderForViewing $orderForViewing */
+        $orderForViewing = $this->getQueryBus()->handle(new GetOrderForViewing($orderId, QuerySorting::DESC));
+
+        $currencyDataProvider = $this->container->get('prestashop.adapter.data_provider.currency');
+        $orderCurrency = $currencyDataProvider->getCurrencyById($orderForViewing->getCurrencyId());
+
+        $formBuilder = $this->get('prestashop.core.form.identifiable_object.builder.cancel_product_form_builder');
+        $backOfficeOrderButtons = new ActionsBarButtonsCollection();
```

Comment:
This is not useful for product list These buttons are used in the top layer of the product page